knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -191,22 +246,32 @@ func installKnativeKafkaChannel(instance *operatorv1alpha1.KnativeKafka, apiclie
 	return nil
 }
 
+// rawKafkaChannelManifest returns KafkaChannel manifest without transformations
+func rawKafkaChannelManifest(apiclient client.Client) (mf.Manifest, error) {
+	return mfc.NewManifest(kafkaChannelManifestPath(), apiclient, mf.UseLogger(log.WithName("mf")))
+}
+
 func kafkaChannelManifest(instance *operatorv1alpha1.KnativeKafka, apiClient client.Client) (*mf.Manifest, error) {
-	manifest, err := mfc.NewManifest(kafkaChannelManifestPath(), apiClient, mf.UseLogger(log.WithName("mf")))
+	manifest, err := rawKafkaChannelManifest(apiClient)
 	if err != nil {
 		return nil, fmt.Errorf("failed to load KafkaChannel manifest: %w", err)
 	}
 
-	manifest, err = manifest.Transform(mf.InjectOwner(instance))
+	transformers := []mf.Transformer{
+		mf.InjectOwner(instance),
+		common.SetOwnerAnnotations(instance.ObjectMeta, common.KafkaOwnerName, common.KafkaOwnerNamespace),
```

Comment:
Impl for `common.SetOwnerAnnitations` exists in a later commit (https://github.com/openshift-knative/serverless-operator/pull/496/commits/359a39b0817b8ccc09c134fa2766efa5c48b447f#diff-052673a614af4880bcfbd6c264d9f495R73). Rebasing messed up stuff.

Reply:
But it is a simple function. Instead of setting the owner annotation with hardcoded keys, it now sets it using the keys passed as arguments.